MonoGame.Framework/Curve.cs

```diff
@@ -3,81 +3,106 @@
 // file 'LICENSE.txt', which is part of this source code package.
 
 using System;
+using System.ComponentModel;
 using System.Runtime.Serialization;
 
 namespace Microsoft.Xna.Framework
 {
-	// TODO [TypeConverter(ExpandableObjectConverter)]
+    /// <summary>
+    /// Contains a collection of <see cref="CurveKey"/> points in 2D space and provides methods for evaluating features of the curve they define.
+    /// </summary>
+    // TODO : [TypeConverter(typeof(ExpandableObjectConverter))]
     [DataContract]
     public class Curve
     {
         #region Private Fields
 
-        private CurveKeyCollection keys;
-        private CurveLoopType postLoop;
-        private CurveLoopType preLoop;
-
-        #endregion Private Fields
+        private CurveKeyCollection _keys;
+        private CurveLoopType _postLoop;
+        private CurveLoopType _preLoop;
```

Comment:
I hate not having underscores on member fields myself, but try to avoid doing "formatting fixes".  We don't need anyone to fix formatting in the code...  focus on docs, bugs, features, etc.  We have plans on fixing formatting in the future.